build(plugin-keychain-memory): fix local imports broken due to .js extensions

### Description

Reproduce by executing this on `main`:

`yarn jest packages/cactus-plugin-keychain-memory/src/test/typescript/integration/api-surface.test.ts`
and then watch it fail as such:

```sh
$ yarn jest packages/cactus-plugin-keychain-memory/src/test/typescript/integration/api-surface.test.ts
jest-haste-map: duplicate manual mock found: ConfigUtil
  The following files share their name; please delete one of them:
    * <rootDir>/packages/cactus-cmd-socketio-server/src/main/typescript/routing-interface/util/__mocks__/ConfigUtil.ts
    * <rootDir>/packages/cactus-cmd-socketio-server/dist/lib/main/typescript/routing-interface/util/__mocks__/ConfigUtil.js

 FAIL  packages/cactus-plugin-keychain-memory/src/test/typescript/integration/api-surface.test.ts
  ● Test suite failed to run

    Cannot find module './default_service_pb.js' from 'packages/cactus-plugin-keychain-memory/src/main/typescript/generated/crpc/services/default_service_connect.ts'

    Require stack:
      packages/cactus-plugin-keychain-memory/src/main/typescript/generated/crpc/services/default_service_connect.ts
      packages/cactus-plugin-keychain-memory/src/main/typescript/plugin-keychain-memory.ts
      packages/cactus-plugin-keychain-memory/src/main/typescript/public-api.ts
      packages/cactus-plugin-keychain-memory/src/test/typescript/integration/api-surface.test.ts

      13 | // @ts-nocheck
      14 |
    > 15 | import { DeleteKeychainEntryV1Request, GetKeychainEntryV1Request, GetPrometheusMetricsV1Response, HasKeychainEntryV1Request, SetKeychainEntryV1Request } from "./default_service_pb.js";
         | ^
      16 | import { DeleteKeychainEntryResponseV1PB } from "../models/delete_keychain_entry_response_v1_pb_pb.js";
      17 | import { Empty, MethodKind } from "@bufbuild/protobuf";
      18 | import { GetKeychainEntryResponseV1PB } from "../models/get_keychain_entry_response_v1_pb_pb.js";

      at Resolver._throwModNotFoundError (node_modules/jest-resolve/build/resolver.js:427:11)
      at Object.<anonymous> (packages/cactus-plugin-keychain-memory/src/main/typescript/generated/crpc/services/default_service_connect.ts:15:1)
      at Object.<anonymous> (packages/cactus-plugin-keychain-memory/src/main/typescript/plugin-keychain-memory.ts:26:1)
      at Object.<anonymous> (packages/cactus-plugin-keychain-memory/src/main/typescript/public-api.ts:4:1)
      at Object.<anonymous> (packages/cactus-plugin-keychain-memory/src/test/typescript/integration/api-surface.test.ts:1:1)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        0.61 s
Ran all test suites matching /packages\/cactus-plugin-keychain-memory\/src\/test\/typescript\/integration\/api-surface.test.ts/i.
```

### Acceptance Criteria
1. Fix the generated code which appears to be importing `.js` extension files **OR** fix the Jest config whichever turns out to be the solution that doesn't add more tech. debt.

--- a/jest.config.js
+++ b/jest.config.js
@@ -3,6 +3,9 @@ module.exports = {
   preset: "ts-jest",
   logHeapUsage: true,
   testEnvironment: "node",
+  moduleNameMapper: {
+    "^(\\.\\.?\\/.+)\\.jsx?$": "$1",
+  },
   maxWorkers: 1,
   maxConcurrency: 1,
   setupFilesAfterEnv: ["jest-extended/all", "./jest.setup.console.logs.js"],

